src/react-hooks/index.ts

```diff
@@ -0,0 +1,7 @@
+import { coreModule } from '../core/module';
+import { buildCreateApi } from '../createApi';
+import { reactHooksModule } from './module';
+
+const createApi = buildCreateApi(coreModule, reactHooksModule);
+
+export { createApi, reactHooksModule };
```

Comment:
Just looking at this part makes it kinda worth it ^^

Reply:
Agreed. This is what caught me.